sermant-plugins/sermant-springboot-registry/config/config.yaml

```diff
@@ -7,6 +7,8 @@ sermant.springboot.registry:
   realmName: www.domain.com
   # Whether to enable traffic statistics, after it is enabled, the traffic entering the plug-in will be printed every time
   enableRequestCount: false
+  # The type of the registry center, currently supports Nacos and Zookeeper
+  registryCenterType:
```

Comment:
Use all uppercase, NACOS or ZOOKEEPER, and registryCenterType should be given default value